static/src/javascripts/boot.js

```diff
@@ -12,20 +12,19 @@ Now we can safely require one of those modules.
 Unfortunately we can't do all of this using the curl API, so we use a
 combination of ajax/eval/curl instead.
 
-Bundles we need to run: commercial + (enhanced-vendor + enhanced)*
-* Only if we detect we should run enhance
+Bundles we need to run: commercial + enhanced
+
+Only if we detect we should run enhance.
  */
 
 define([
     'Promise',
```

Comment:
Yeah, the thought did cross my mind. If you agree it's worth doing, I'll open another pr for that?